### PR TITLE
Application interface

### PR DESCRIPTION
**Pull request by MarcosCGlobant**
Application interface configuration, color scheme and constraints added to the buttons and label.

Screenshot_1582895884

### CODE REVIEW

**Review by BQuinteros — APPROVED** [review]

File: app/app/src/main/res/layout-v21/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    android:layout_margin="@dimen/activity_layout_margin"
+    android:background="@color/main_background"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/calculation_label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/label_background"
+        android:text="@string/_0"
```
Comment (mauroLonghi) [inline app/app/src/main/res/layout-v21/activity_main.xml]: Pls, use another representative name for this element

File: app/app/src/main/res/layout-v21/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        android:text="@string/clear"
+        android:backgroundTint="@color/action_button_color"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/calculation_label" />
+
+    <Button
+        android:id="@+id/number_one"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/_1"
```
Comment (mauroLonghi) [inline app/app/src/main/res/layout-v21/activity_main.xml]: same, and in all the number's please.

File: app/app/src/main/res/layout-v21/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        android:backgroundTint="@color/button_color"
+        app:layout_constraintEnd_toStartOf="@id/number_three"
+        app:layout_constraintTop_toBottomOf="@id/calculation_label" />
+
+    <Button
+        android:id="@+id/number_three"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/_3"
+        android:backgroundTint="@color/button_color"
```
Comment (mauroLonghi) [inline app/app/src/main/res/layout-v21/activity_main.xml]: Take a look, you use the same width, height and backgroundTint for all number buttons, maybe you can use a "style" to re-use this properties.

File: app/app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,10 +1,168 @@
...
+    android:layout_margin="@dimen/activity_layout_margin"
+    android:background="@color/main_background"
     android:orientation="vertical">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
+    <TextView
+        android:id="@+id/calculation_label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="#F6FBF9"
```
Comment (mauroLonghi) [inline app/app/src/main/res/layout/activity_main.xml]: Pls, use a resource to remove this hardcode color.

File: app/app/src/main/res/layout-v21/activity_main.xml (hunk 1 above)
Comment (MarcosCGlobant) [reply app/app/src/main/res/layout-v21/activity_main.xml]: done, thanks

File: app/app/src/main/res/layout-v21/activity_main.xml (hunk 2 above)
Comment (MarcosCGlobant) [reply app/app/src/main/res/layout-v21/activity_main.xml]: done, thanks

**Review by BQuinteros — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**